__test__/relation-tests.js

```diff
@@ -44,6 +44,30 @@ test('Test CREATE relation', (assert) => {
     });
 });
 
+
+
+test('Test MERGE relation', (assert) => {
+    let node1 = nodes[0];
+    let node2 = nodes[1];
+    let rel1 = OGMNeoRelation.relateMerge(node1.id, 'relatedto', node2.id, { property: 'a' });
+    let rel2 = OGMNeoRelation.relateMerge(node1.id, 'relatedto', node2.id, {});
+    let rel3 = OGMNeoRelation.relateMerge(node1.id, 'relatedto', node2.id, { property: 'a' });
```

Comment:
rel3 and rel4 aren't being used here on this test

Reply:
Alright I forgot about this. I'll fix it shortly.